diesel_infer_schema/src/inference.rs

```diff
@@ -56,15 +56,17 @@ pub(crate) fn establish_connection(database_url: &str) -> Result<InferConnection
                 It must start with postgres:// or postgresql://",
                 database_url,
             ).into())
-        }
+        },
         #[cfg(all(feature = "mysql", not(any(feature = "sqlite", feature = "postgres"))))]
         _ => {
             Err(format!(
                 "{} is not a valid MySQL database URL. \
                 It must start with mysql://",
                 database_url,
             ).into())
-        }
+        },
+        #[cfg(not(any(feature = "mysql", feature = "sqlite", feature = "postgres")))]
+        _ => compile_error!("No database backend specified."),
```

Comment:
Since this message is user-facing, can we provide some information about how to fix this? It should mention that `features = ["postgres"]`, etc are required.